webcompat/static/css/development/base/variable.css

```diff
@@ -1,47 +0,0 @@
-/*----------Variable----------*/
```

Comment:
Just to understand, variables are now defined in a `root` selector in each "module"?